examples/example01-car-trade/validatorDriver/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "webapi",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc --watch"
+  },
+  "devDependencies": {
+    "typescript": "^3.9.3"
+  },
+  "dependencies": {
```

Comment:
@takeutak Shouldn't this use the existing cactus npm packages as dependencies? 
My concern: If the example writes everything from scratch then it's an example of how to create a piece of software like Cactus, not an example of how to use Cactus the software that was already written by someone.